userguide/jails.rst

```diff
@@ -149,8 +149,8 @@ specified RELEASE directories as nullfs mounts over the jail
 directories. Basejails are not linked to the original RELEASE when
 upgraded.
 
-Choose the version of FreeBSD to install for this jail. The FreeBSD
-version cannot be newer than the FreeBSD version used by %brand%.
+Jails can run FreeBSD versions up to the same version as the host
+%brand% system. Newer releases are not shown.
```

Comment:
```suggestion
%brand% system.
```

Reply:
That sentence was recommended to be added by Warren, so I'm hesitant to remove it...

Reply:
Fair enough. Just seems like if it's not shown, then there's no need to mention that?